themes/mlibrary/custom.php

```diff
@@ -13,6 +13,23 @@
  * Extends featured exhibit function to include snippet from description for exhibits and
  * read more link
  **/
+
+/**
+* Create the information in cards in introduction page, image with title and text
+*
+**/
+function mlibrary_display_exhibit_card_info($rawAttachment,$block,$exhibitPage)
+{
+   $file = $rawAttachment[0]->getFile();
+   $page_image = file_image('thumbnail','', $file);
+   $page_title = metadata($exhibitPage, 'title').'<br>';
+   $page_description = snippet_by_word_count(metadata($block[0], 'text',array('no_escape' => true)),20,'..');
+
+   $page_card_info = $page_image.$page_title.$page_description;
+
+   return $page_card_info;
```

Comment:
I recommend merging 28 and 30 and eliminating the `$page_card_info variable`.

Reply:
I don't think this was what I had meant, though if you prefer returning an array that's fine.  I meant:

```
$page_card_info = $page_image.$page_title.$page_description;

return $page_card_info;
```
Could be:

```
return $page_image.$page_title.$page_description;
```